### DISCUSSION TITLE
Error: neovim: no bottle available!

### DISCUSSION THREAD

**Discussion by lanox** [Homebrew/discussions#1034] [Everyday usage] [answered]
Hi All

I am not sure if this is the right place to ask but I am having the following issues when trying to upgrade neovim from HEAD.

```
brew upgrade neovim --fetch-HEAD
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 2 formulae.

==> Upgrading 1 outdated package:
neovim HEAD-097ec71_1 -> HEAD-dc8273f_1
==> Upgrading neovim HEAD-097ec71_1 -> HEAD-dc8273f_1
Error: neovim: no bottle available!
You can try to install from source with:
  brew install --build-from-source neovim
Please note building from source is unsupported. You will encounter build
failures with some formulae. If you experience any issues please create pull
requests instead of asking for help on Homebrew's GitHub, Twitter or any other
official channels.
```

I was wondering if someone can help me out.
Thanks

**Comment by carlocab** [comment, marked as answer]
Trying adding a `--build-from-source` flag (equivalent: `-s`) to your `brew upgrade` command.

**Reply by lanox** [reply]
@carlocab I saw that but, it also says it's unsupported, should I still be doing that? and in past, I didn't need to do that?

**Reply by carlocab** [reply]
`HEAD` installs are also unsupported, as it requires building from source. If you're still fine with building from source, then you can pass the flag.

**Reply by lanox** [reply]
Thanks, understood.
